app/models/twitter.py

```diff
@@ -19,6 +21,7 @@ class Tweet(EmbeddedDocument):
     tweet_id = StringField(primary_key=True)
     text: str = StringField(required=True)
     created_at: datetime = DateTimeField(required=True)
+    referenced_tweets: tweepy.ReferencedTweet = ListField()
```

Comment:
Is this a single tweet or a list of referenced tweets?

Reply:
@phhai-hcmut in the docs, it said "A list of Tweets this Tweet refers to"

Reply:
But the type of `referenced_tweets` you specified here is just `tweepy.ReferencedTweet`

Reply:
the type of `tweepy.ReferencedTweet` return a list itself, i think. I tried to wrap it inside a `List[]` but it got wrong

Reply:
[This](https://github.com/tweepy/tweepy/blob/v4.4.0/tweepy/tweet.py#L76) say otherwise

Reply:
fight fight fight 🍾

Reply:
i got this when print itself and type
```shell
<class 'list'>
[<ReferencedTweet id=1507148569513537540 type=quoted]
```

Reply:
I means the correct type for `referenced_tweets` is `List[tweepy.ReferencedTweet]`

Reply:
let me try again :V

Reply:
fixed